source/crud/tests/v2/default-write-concern.yml

```diff
@@ -0,0 +1,413 @@
+data:
+  - _id: 1
+    x: 11
+  - _id: 2
+    x: 22
+collection_name: default_write_concern_coll
+database_name: default_write_concern_db
+tests:
+  - description: Aggregate with $merge
+    operations:
+      - object: collection
+        name: aggregate
+        arguments:
+          pipeline: &ref_0
+            - $match:
+                _id:
+                  $gt: 1
+            - $merge:
+                into: other_test_collection
+    expectations:
+      - command_started_event:
+          command:
+            aggregate: default_write_concern_coll
+            pipeline: *ref_0
+            writeConcern: null
+    outcome:
+      collection:
+        name: other_test_collection
+        data:
+          - _id: 2
+            x: 22
+  - description: Aggregate with $out
+    operations:
+      - object: collection
+        name: aggregate
+        arguments:
+          pipeline: &ref_1
+            - $match:
+                _id:
+                  $gt: 1
+            - $out: other_test_collection
+          batchSize: 2
+    outcome:
+      collection:
+        name: other_test_collection
+        data:
+          - _id: 2
+            x: 22
+    expectations:
+      - command_started_event:
+          command:
+            aggregate: default_write_concern_coll
+            pipeline: *ref_1
+            writeConcern: null
+  - description: runCommand with a write command
+    operations:
+      - object: database
+        name: runCommand
+        arguments:
+          command:
+            create: test_collection
+    expectations:
+      - command_started_event:
+          command:
+            create: test_collection
+            writeConcern: null
+  - description: create and drop index
+    operations:
+      - object: collection
+        name: createIndex
+        arguments:
+          model:
+            x: 1
```

Comment:
Is this the first time that we are testing createIndex and createIndexes? What do you think about defining the argument format in the readme? I have no preference between the two argument forms presented here (`model: {x:1}` and  `{name: _x_1, key: {x:1}}`) but I do think we should standardize on one of them.

Reply:
Assuming this refers to the [Index Management](https://github.com/mongodb/specifications/blob/master/source/index-management.rst) spec, I'd suggest we stick to the `createIndex(keys: Document, options: IndexOptions)` API here. I think the `model` API is less common.

Reply:
Changed to `keys`. I did not need to specify any index options for this test since it assumes the default `x_1` index name.